pages/interoperability.html

```diff
@@ -1,273 +1,263 @@
-    </div><!-- end footer -->
-    <!--
-           ==================================================
-       -->
-    <!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
-
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/js/bootstrap.min.js"></script>
-
-
-    <script type="text/javascript" src="js/owl.carousel.js"></script><!-- Owl Carousel Plugin -->
-
-    <script type="text/javascript" src="js/SmoothScroll.js"></script>
-
-    <!-- Google Map -->
-    <script type="text/javascript" src="https://maps.googleapis.com/maps/api/js?key=AIzaSyASm3CwaK9qtcZEWYa-iQwHaGi3gcosAJc&amp;sensor=false"></script>
-    <script type="text/javascript" src="js/map.js"></script>
-
-    <!-- Parallax Effects -->
-    <script type="text/javascript" src="js/skrollr.js"></script>
-    <script type="text/javascript" src="js/imagesloaded.js"></script>
-
-    <!-- Portfolio Filter -->
-    <script type="text/javascript" src="js/jquery.isotope.js"></script>
-
-    <!-- LightBox Nivo -->
-    <script type="text/javascript" src="js/nivo-lightbox.min.js"></script>
-
-    <!-- Contact page-->
-    <script type="text/javascript" src="js/jqBootstrapValidation.js"></script>
-    <script type="text/javascript" src="js/contact.js"></script>
-
-    <!-- Javascripts
-    ================================================== -->
-    <script type="text/javascript" src="js/main.js"></script>
-    <script type="text/javascript" src="js/menu.js"></script>
-
-</body>
+﻿<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8" />
+    <title>AHIE - African Health Information Exchange</title>
+    <meta name="description" content="African Health Information Exchange -Technology in support of health services for patients with HIV and TB">
+    <meta name="keywords" content="health, HIV, TB, health information, information exchange, interoperability">
+    <meta name="author" content="AHIE.net">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta content="width=device-width, initial-scale=1" name="viewport" />
+    <meta name="format-detection" content="telephone=no">
+
+
+    <!-- Favicons
+    ================================================== -->
+    <link href="/apple-touch-icon.png" rel="apple-touch-icon" sizes="180x180" />
+    <link href="/favicon-32x32.png" rel="icon" sizes="32x32" type="image/png" />
+    <link href="/favicon-16x16.png" rel="icon" sizes="16x16" type="image/png" />
+    <link href="/site.webmanifest" rel="manifest" />
+    <link color="#b30e19" href="/safari-pinned-tab.svg" rel="mask-icon" />
+    <meta name="msapplication-TileColor" content="#da532c">
+    <meta name="theme-color" content="#ffffff">
+    <!-- Bootstrap -->
+    <link rel="stylesheet" href="css/font-awesome.min.css">
+
+    <!-- fonts, styles and functions -->
+    <link href="fonts/font-style.css" rel="stylesheet">

```

Comment:
I can't where  `fonts/font-style.css` is supposed to be loaded from

Reply:
I've removed this code from all files since this font-style.css does not exist

Reply:
I don't yet see the changes in the commit history: https://github.com/wc-phdc-hia/ahie-website/pull/2/commits